subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf52840.h

```diff
@@ -404,16 +404,25 @@ static inline u32_t hal_radio_phy_mode_get(u8_t phy, u8_t flags)
 	case BIT(0):
 	default:
 		mode = RADIO_MODE_MODE_Ble_1Mbit;
+
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
 		/* Workaround: nRF52840 Engineering A Errata ID 164 */
 		*(volatile u32_t *)0x4000173c &= ~0x80000000;
+#endif /* CONFIG_BT_CTLR_PHY_CODED */
+
 		break;
 
 	case BIT(1):
 		/* Workaround: nRF52840 Engineering A Errata ID 164 */
+
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
```

Comment:
Same question here, as above.
Also, could you improve style consistency with the above "case"?
E.g in both "case" we should have:
1. Workaround and comment of workaround inside #ifdef 
2. mode = ....

Reply:
What happened here! My bad, I should slow down my rate of PRs per day!

Will fix this, thank you for a thorough review.